### PR TITLE
Setting for comment style when disabling a line

### PR DESCRIPTION
**Pull request by MariaSolOs**
Adds support for the feature requested in #1490.

### CODE REVIEW

**Comment by dbaeumer** [comment]
@MariaSolOs should we first get https://github.com/microsoft/vscode-eslint/pull/1506 in and then use the new method `getDisableRuleEditInsertionIndex`. Looks like this PR somehow conflicts a little with the other one.

**Comment by MariaSolOs** [comment]
> @MariaSolOs Maria Solano FTE should we first get #1506 in and then use the new method `getDisableRuleEditInsertionIndex`. Looks like this PR somehow conflicts a little with the other one.

Absolutely, we should merge the bug-fix first and I'll rebase this PR since otherwise we'll have a merge conflict for sure.

**Comment by MariaSolOs** [comment]
Alright, this PR was rebased and now the changes from that bug fix have been included as well! :D

**Comment by MariaSolOs** [comment]
Any updates about this? :)

File: client/src/client.ts [hunk 1]
```diff
@@ -617,7 +617,7 @@ export namespace ESLintClient {
 					workingDirectory: undefined,
 					workspaceFolder: undefined,
 					codeAction: {
-						disableRuleComment: config.get('codeAction.disableRuleComment', { enable: true, location: 'separateLine' as 'separateLine' }),
+						disableRuleComment: config.get('codeAction.disableRuleComment', { enable: true, location: 'separateLine' as 'separateLine', commentStyle: 'line' as 'line' }),
```
Comment (dbaeumer) [inline client/src/client.ts:620]: We should use the new TS syntax here which is `'line' as const` and should apply the same for 'separateLine'

**Review by dbaeumer — CHANGES_REQUESTED** [review]
Looks great. Only one small comment.

**Review by dbaeumer — APPROVED** [review]

**Review by aeschli — APPROVED** [review]

**Pull request merged**